content/en/docs/ambient/overview/APIs/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Ambient APIs
+description: Understanding ambient APIs and its status.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+This page lists all the user facing APIs relevant to the ambient mode and its feature status.
+
+## Labels
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `istio.io/dataplane-mode` | Beta |
+| `istio.io/use-waypoint` | Beta |
+| `istio.io/use-waypoint-namespace` | Beta |
+| `istio.io/waypoint-for` | Beta |
+
+TODO: add link to labels page for each when 3307 is merged.
+
+## Annotations
+
+Below are annotations that can be used to customize your waypoint proxies:
+
+| Name | Status |
+| ------------------------------- | -------------------------- |
+| `gateway.istio.io/service-account` | Alpha |
+| `ambient.istio.io/waypoint-inbound-binding` | Alpha |
+| `gateway.istio.io/name-override` | Alpha |
+
+TODO: add link to annotations page for each when 3307 is merged.
+
+## Kubernetes Gateway API resources
+
+Below are the resources you can use to configure your Ingress gateway or waypoint proxies:
+
+|  Name  | Status |
+| --- | --- |
+| [`HTTPRoute`](https://gateway-api.sigs.k8s.io/guides/http-routing/) | Beta |
```

Comment:
HTTPRoute is GA and so is Istio's support for it generally. If this is documentation for Ambient GA, shouldn't this status be GA as well?

Reply:
Updated!